spec/spec_helper.rb

```diff
@@ -45,7 +45,7 @@ def copy_fixture_to_pod(name, pod)
 VCR.configure do |c|
   c.cassette_library_dir = ROOT + 'spec/fixtures/vcr_cassettes'
   c.hook_into :webmock
-  c.ignore_hosts 'codeclimate.com'
+  c.ignore_hosts 'codeclimate.com', 'some-url'
```

Comment:
I dont think this host should be ignored?

Reply:
You're right.

I was thinking that if you forget to use `WebMock` to mock an endpoint, then `VCR` shouldn't try to handle it since it's not a legitimate URL to begin with and will never be recorded as a cassette. However, it's better that `VCR` complains about it than for `Net::HTTP` to try to connect to it.